awjf_pub/awjf_pub_sw_funktion_ar.sql

```diff
@@ -0,0 +1,3 @@
+SELECT ogc_fid,fk,fr,name,sturz,rutsch,grs,lawine,anderekt,obj_kat,schaden_po,h_gef_pot,igef_pot,bemerkunge,flaeche,st_multi(wkb_geometry) as wkb_geometry,status,name_2,gem_name,gem_bfs
```

Comment:
Renaming ogc_fid to t_id and wkb_geometry to geometrie
better split the attribute listing on two lines